packages/react-catalog-view-extension/src/components/CatalogTile/__snapshots__/CatalogTile.test.tsx.snap

```diff
@@ -65,28 +65,30 @@ exports[`CatalogTile renders properly 1`] = `
             class="catalog-tile-pf-badge-container"
           >
             <span>
-              <span
-                class="catalog-tile-pf-badge"
-              >
-                <svg
-                  aria-hidden="true"
-                  fill="currentColor"
-                  height="1em"
-                  role="img"
-                  style="vertical-align: -0.125em;"
-                  viewBox="0 0 512 512"
-                  width="1em"
-                >
-                  <path
-                    d="M487.4 315.7l-42.6-24.6c4.3-23.2 4.3-47 0-70.2l42.6-24.6c4.9-2.8 7.1-8.6 5.5-14-11.1-35.6-30-67.8-54.7-94.6-3.8-4.1-10-5.1-14.8-2.3L380.8 110c-17.9-15.4-38.5-27.3-60.8-35.1V25.8c0-5.6-3.9-10.5-9.4-11.7-36.7-8.2-74.3-7.8-109.2 0-5.5 1.2-9.4 6.1-9.4 11.7V75c-22.2 7.9-42.8 19.8-60.8 35.1L88.7 85.5c-4.9-2.8-11-1.9-14.8 2.3-24.7 26.7-43.6 58.9-54.7 94.6-1.7 5.4.6 11.2 5.5 14L67.3 221c-4.3 23.2-4.3 47 0 70.2l-42.6 24.6c-4.9 2.8-7.1 8.6-5.5 14 11.1 35.6 30 67.8 54.7 94.6 3.8 4.1 10 5.1 14.8 2.3l42.6-24.6c17.9 15.4 38.5 27.3 60.8 35.1v49.2c0 5.6 3.9 10.5 9.4 11.7 36.7 8.2 74.3 7.8 109.2 0 5.5-1.2 9.4-6.1 9.4-11.7v-49.2c22.2-7.9 42.8-19.8 60.8-35.1l42.6 24.6c4.9 2.8 11 1.9 14.8-2.3 24.7-26.7 43.6-58.9 54.7-94.6 1.5-5.5-.7-11.3-5.6-14.1zM256 336c-44.1 0-80-35.9-80-80s35.9-80 80-80 80 35.9 80 80-35.9 80-80 80z"
-                  />
-                </svg>
+              <div>
```

Comment:
due to the extra div wrapper to get a ref, there's an additional level of nesting. Wonder if that could spell trouble...

Reply:
what sort of trouble do you envision, @jschuler 
I was thinking about that and figured the primary problem would be if people wrote relative styles or wrote unit tests with relative query selectors - which I believe is not best practice... But is that something we should avoid. Is there another reason you can imagine?

Reply:
Yes, there could be style issues, and if a project does snapshot testing it could also invalidate them if they use a popper based component I think. I'm just wondering if there's a way to avoid the extra nesting? Maybe through a combination of cloning the element and providing a ref to the cloned element?

Reply:
Snapshots would have to be updated yeah. Also like in our integration tests I had to update some selectors.

The issue I can foresee with cloning the element and passing in a different ref is that some components use internal refs for their behavior and I'm not sure if that would mess with anything?

Reply:
good point @jschuler I think that would constitute a breaking change.